Store/admin/components/Account/Details.php

```diff
@@ -143,7 +144,7 @@ protected function getOrdersTableModel(SwatTableView $view)
 	// }}}
 	// {{{ protected function getAddressesTableModel()
 
-	protected function getAddressesTableModel(SwatTableView $view)
+	protected function getAddressesTableModel(SwatTableView $view): SwatTableStore
```

Comment:
Using the more generic interface `SwatTableModel` makes sense here

Reply:
See above. :)